app/src/test/java/org/simple/clinic/sync/DataSyncTest.kt

```diff
@@ -38,26 +38,16 @@ class DataSyncTest {
       syncGroup = SyncGroup.FREQUENT
   )
 
-  private val dailySyncConfig = SyncConfig(
-      syncInterval = SyncInterval(
-          frequency = Duration.ofMinutes(16),
-          backOffDelay = Duration.ofMinutes(5)
-      ),
-      pullBatchSize = 10,
-      pushBatchSize = 10,
-      syncGroup = SyncGroup.DAILY
-  )
-
   @Test
-  fun `when syncing everything, all the model syncs should be invoked`() {
+  fun `when syncing, all the model syncs should be invoked`() {
     val modelSync1 = FakeModelSync(
         _name = "sync1",
-        config = dailySyncConfig
+        config = frequentSyncConfig
```

Comment:
So we'll be removing `SyncInterval` and `SyncGroup` from the FakeModelSync when we finish removing frequent type as well, right?

Reply:
Yup, that is correct.